modules/prebidServerBidAdapter/config.js

```diff
@@ -1,27 +1,18 @@
 // accountId and bidders params are not included here, should be configured by end-user
 export const S2S_VENDORS = {
-  'appnexus': {
```

Comment:
this is used by some [integration examples and unit tests](https://github.com/search?q=repo%3Aprebid%2FPrebid.js+defaultVendor%3A+%27appnexus%27&type=code), those should probably also be updated.

Reply:
Pushed a change to get the remaining references